src/Mod/PartDesign/Gui/TaskHoleParameters.ui

```diff
@@ -69,7 +69,7 @@
    <item row="3" column="1">
     <widget class="QCheckBox" name="ModelActualThread">
      <property name="enabled">
-      <bool>false</bool>
+      <bool>true</bool>
```

Comment:
Please don't turn it on by default. Real threads are an expert feature consuming computation bandwidth.

Reply:
It is not 'Checked' only enabled. I.e checkable. Default is and should be unchecked (as you say)

Reply:
Thanks.